language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/codegen/ContractId.java

```diff
@@ -54,7 +54,7 @@ public final ExerciseCommand makeExerciseCmd(String choice, Value choiceArgument
 
   // overridden by every code generator, but decoding abstractly can e.g.
   // produce a ContractId<Foo> that is not a Foo.ContractId
-  protected ContractTypeCompanion getCompanion() {
+  protected ContractTypeCompanion<T, T> getCompanion() {
```

Comment:
```suggestion
  protected ContractTypeCompanion<T, ?> getCompanion() {
```

`T` in the second position is wrong for interfaces.